--- helper	2025-01-29 13:06:07.262667754 -0800
+++ helper_new	2025-01-29 13:06:26.622667509 -0800
@@ -28,7 +28,7 @@
     CONTAINER_ID=$(docker ps -a -q --filter "name=project1_instance")
     if [ -z "$CONTAINER_ID" ]; then
         echo "No existing container found. Creating a new one..."
-        docker create --name project1_instance --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v $(pwd)/project:/autograder/submission -v $(pwd)/results:/autograder/results --entrypoint tail "$NAME" -f /dev/null
+        docker create --name project1_instance --network host --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v $(pwd)/project:/autograder/submission -v $(pwd)/results:/autograder/results --entrypoint tail "$NAME" -f /dev/null
         CONTAINER_ID=$(docker ps -a -q --filter "name=project1_instance")
     fi